lib/test.js

```diff
@@ -22,6 +22,7 @@ function Test(title, fn) {
   Runnable.call(this, title, fn);
   this.pending = !fn;
   this.type = 'test';
+  this.body = (fn || '').toString();
```

Comment:
Same reason why `Test#clone` doesn't apply `this.type = 'test';`. It's already handled in the constructor :)